docresources/SyncBot/syncbot-readme.md

```diff
@@ -265,6 +265,10 @@ Errors, where possible, are reported in the originating thread as whispers.
 		* Add the flow mapping to the array
 	* `SYNCBOT_LISTENER_CONSTRUCTORS.front.channelPerInbox`
 		* Add the map of inbox to channel to the array
+1) Add the inbox to the `SyncBot <- Inboxes` rule, settings should be as follow:
+	* When `Inbound message`, `Outbound message`, `Outbound reply`, `Comment` and `Mention`
```

Comment:
I get the gist, but I think the bullet points are confusing as it implies these are three separate things. Am I misunderstanding something, or is this not a test and result order?

Reply:
![front](https://user-images.githubusercontent.com/20857369/36679649-a47dac6c-1b0b-11e8-9f50-57b60d622a92.png)

Reply:
Understand that, but if it's going to be written in documentation, possibly this is better:
```
Add an appropriate rule:
WHEN: <blah>
IF <at least one> CONDITION IS MET:
  - <condition one>
  - <condition two>
THEN: Send a Webhook: <blah>
```
Personally, to me, this mimics the UI far better than three bullet points.